crabs/detection_tracking/train_model.py

```diff
@@ -36,6 +36,7 @@ class DectectorTrain:
     """
 
     def __init__(self, args):
+        self.args = args
```

Comment:
fantastic additions! 🚀 

Just about the syntax:
I was wondering if we could have something more descriptive than `args`. Maybe:
```
 def __init__(self, args):
        [...]
	self.debug_fast_dev_run = args.fast_dev_run
	self.debug_limit_train_batches = args.limit_train_batches
	self.load_config_yaml()
```

Pros: it makes it more explicit what we are passing (hence more readable), it would be consistent with the rest of the `args` we have, and we avoid repetition (as it is now, we have `self.args.accelerator` and `self.accelerator` right?). Cons: less flexible when we add new args. But I think pros outweight cons.